parts/linux/cloud-init/artifacts/components.json

```diff
@@ -3,49 +3,58 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/autoscaler/addon-resizer:*",
       "amd64OnlyVersions": [],
-      "multiArchVersions": [
-        "1.8.20",
-        "1.8.22"
+      "multiArchVersionsV2": [
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com,name=oss/kubernetes/autoscaler/addon-resizer",
```

Comment:
can we put a space after `, `

`"registry=https://mcr.microsoft.com, name=oss/kubernetes/autoscaler/addon-resizer",`

Reply:
done